commands/team_e2e_test.go

```diff
@@ -51,3 +53,40 @@ func (s *MmctlE2ETestSuite) TestRenameTeamCmdF() {
 		s.Equal("Cannot rename team '"+s.th.BasicTeam.Name+"', error : : You do not have the appropriate permissions., ", err.Error())
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestListTeamsCmdF() {
+	s.SetupTestHelper().InitBasic()
+	mockTeamName := "mockteam" + model.NewId()
+	mockTeamDisplayname := "mockteam_display"
+	_, err := s.th.App.CreateTeam(&model.Team{Name: mockTeamName, DisplayName: mockTeamDisplayname, Type: model.TEAM_OPEN, DeleteAt: 1})
+	s.Require().Nil(err)
+
+	s.RunForSystemAdminAndLocal("Should print list of teams", func(c client.Client) {
+		printer.Clean()
+
+		err := listTeamsCmdF(c, &cobra.Command{}, []string{})
+		s.Require().Nil(err)
+		s.Len(printer.GetLines(), 2)
+		team := printer.GetLines()[0].(*model.Team)
+		s.Equal(s.th.BasicTeam.Name, team.Name)
+	})
+
+	s.RunForSystemAdminAndLocal("Should print archived teams", func(c client.Client) {
+		printer.Clean()
+		printer.SetFormat(printer.FormatPlain)
+		defer printer.SetFormat(printer.FormatJSON)
+
+		err := listTeamsCmdF(c, &cobra.Command{}, []string{})
+		s.Require().Nil(err)
+		s.Len(printer.GetLines(), 2)
+		s.Equal(printer.GetLines()[1], mockTeamName+" (archived)")
```

Comment:
Actually, both calls use the same command, so I'd suggest to assert both as part of the first test case and change its name to something like "Should print both active and archived teams for syasdmin and local clients"

Reply:
Thanks for the feedback - I compressed both cases into one using the model struct instead of checking the text output.